newsroom/templates/layout.html

```diff
@@ -81,10 +81,10 @@
         {% endblock %}
     </div>
     <footer class="footer">
-        <a href="https://www.aap.com.au/legal/" target="_blank">Privacy Policy</a>
-        <a href="https://www.aap.com.au/legal/" target="_blank">Terms and Conditions</a>
-        <a href="https://www.aap.com.au/contact/sales-inquiries/" target="_blank">Contact Us</a> 
-        <a href="#">Copyright</a>
+        <a href="https://www.aap.com.au/legal/" target="_blank">{{ gettext('Privacy Policy') }}</a>
```

Comment:
pls put put this into a `footer.html` template so we can customize it via themes easily